pick.c

```diff
@@ -111,6 +112,7 @@ static int			 descriptions;
 static int			 sort = 1;
 static int			 use_alternate_screen = 1;
 static int			 use_keypad = 1;
+static int			 vi_mode = 0;
```

Comment:
No need to initialize a static variable to 0. Might be done for clarity, though.

Reply:
Right ... fixed ...